### PR TITLE
Enabling so that we can get logs in json

### PR DESCRIPTION
**Pull request by mattiashem**

### CODE REVIEW

File: main.sh [hunk 1]
```diff
@@ -25,6 +25,13 @@ if [ "${SEC_PRCE_MATCH_LIMIT_RECURSION}" != "" ]; then
   echo "SecPcreMatchLimitRecursion set to '${SEC_PRCE_MATCH_LIMIT_RECURSION}'"
 fi
 
+
+if [ "${LOG_JSON}" != "" ]; then
```
Comment (ollemuhr) [inline main.sh]: What if LOG_JSON=false ?

File: main.sh [hunk 2]
```diff
@@ -25,6 +25,13 @@ if [ "${SEC_PRCE_MATCH_LIMIT_RECURSION}" != "" ]; then
...
+  echo "SecAuditLogFormat json" >>  /etc/modsecurity.d/modsecurity.conf
```
Comment (ollemuhr) [inline main.sh]: Ah, it was this easy :)

**Comment by Nemie** [comment]
So is this OK to merge so we can build a new version and try it out?

**Comment by ysoldak** [comment]
Generalised a bit and added a way to set `SecAuditLogType`.
People say shall set `SecAuditLogType Parallel` for json audit logs to work, see
https://github.com/SpiderLabs/ModSecurity-nginx/issues/47

**Pull request merged**